### DISCUSSION TITLE
Paypal link not working?

### DISCUSSION THREAD

**Discussion by rc3105** [ts1/BLEUnlock#119] [General]
Hey there, love the app.

I'd been wondering how to do this to unlock my front door and wow all the hard work is already done!

Thank you so much.

I'm using the event script to fire off an iOt command so the lock only needs a button mash instead of the whole combination punched in.  Soooo nice when my arms are full of groceries or some such.

On to the main topic, I tried to send a few bucks via paypal and the link didn't work.  After following the Sponsor button it took me to a PayPal page that works, but then errors out that the account can't receive any more funds until "more information is supplied".  

I don't know what's going on with paypal but you might want to look into it if you were not already aware of the issue.

**Comment by ts1** [comment]
Thank you for your support!
PayPal link is not working for some reason.
I deleted the link.
If it's okay with you, please consider supporting via Buy Me a Coffee (or lots of coffee!)
